### ISSUE TITLE
0.97 necrobot dont work with pogolocation 1.1.0

### ISSUE THREAD

**Issue by luigidanield2** [NoxxDev/NecroBot#749]
hi the sniping dont work in necrobot 0.97 i use snipelocation server true but he snipe one pokemon and after stop dont catch another from to list of po location feeder

**Comment by iwean** [comment]
Are you set true to Websocket? 
And the port are the same?

**Comment by iwean** [comment]
For me work fine...
....
![screenshot_159](https://cloud.githubusercontent.com/assets/4365551/18128151/94c34520-6fae-11e6-9001-7e11bdb20431.jpg)

**Comment by luigidanield2** [comment]
no me don't work

**Comment by FureszOszkar** [comment]
It works, check your configs! (the BOT and the feeder)

**Comment by avntty** [comment]
Bang iwan kalo pakai snipe ky gitu aman ng sih, soalnya terakhir sy pake itu malah kna ban perm.

**Comment by samuraitruong** [comment]
still using long distance snipe :) will get ban soon when you reach 29, 30

**Comment by luigidanield2** [comment]
i don't understand!!!!!i can't  snipe location server ????with pogo location feder???i must use only with necrobot sniper???please send me your configuaration

**Comment by avntty** [comment]
yeah i got ban less than a week after snipe long distance.  now i start play again from lvl 1 RIP...LOL

**Comment by iwean** [comment]
Ga ada yg aman mas @avntty ...nyari fun nya aja... ban satu tumbuh seribu... hehehe

**Comment by avntty** [comment]
agree

**Comment by luigidanield2** [comment]
can i have your configuration here please???????

**Comment by iwean** [comment]
NecroBot:
  "WebsocketsConfig": {
    "UseWebsocket": true,
    "WebSocketPort": 14251
  },

PogoLocationFeeder:
  "BotWebSocketPorts": [
    14251
  ],

**Comment by darkfriend77** [comment]
sniping with location feeder isn't working ...

`  "SnipeConfig": {
    "UseSnipeLocationServer": true,
    "SnipeLocationServer": "localhost",
    "SnipeLocationServerPort": 16969,
    "GetSniperInfoFromPokezz": false,
    "GetOnlyVerifiedSniperInfoFromPokezz": false,
    "GetSniperInfoFromPokeSnipers": false,
    "GetSniperInfoFromPokeWatchers": false,
    "GetSniperInfoFromSkiplagged": false,
    "MinPokeballsToSnipe": 20,
    "MinPokeballsWhileSnipe": 0,
    "MinDelayBetweenSnipes": 60,
    "SnipingScanOffset": 0.005,
    "SnipeAtPokestops": true,
    "SnipeIgnoreUnknownIv": false,
    "UseTransferIvForSnipe": true,
    "SnipePokemonNotInPokedex": true,
    "UseSnipeLimit": false,
    "SnipeRestSeconds": 120,
    "SnipeCountLimit": 39
  },`

`  "PokemonToSnipe": {
    "Locations": [],
    "Pokemon": [
      "venusaur",
      "charizard",
      "blastoise",
      "beedrill",
      "raichu",
      "sandslash",
      "nidoking",
      "nidoqueen",
      "clefable",
      "ninetales",
      "golbat",
      "vileplume",
      "golduck",
      "primeape",
      "arcanine",
      "poliwrath",
      "alakazam",
      "machamp",
      "golem",
      "rapidash",
      "slowbro",
      "farfetchd",
      "muk",
      "cloyster",
      "gengar",
      "exeggutor",
      "marowak",
      "hitmonchan",
      "lickitung",
      "rhydon",
      "chansey",
      "kangaskhan",
      "starmie",
      "mrMime",
      "scyther",
      "magmar",
      "electabuzz",
      "jynx",
      "gyarados",
      "lapras",
      "ditto",
      "vaporeon",
      "jolteon",
      "flareon",
      "porygon",
      "kabutops",
      "aerodactyl",
      "snorlax",
      "articuno",
      "zapdos",
      "moltres",
      "dragonite",
      "mewtwo",
      "mew"
    ]
  },`


**PogoLocationFeeder\v0.1.10\Config**
`{
  "Port": 16969,
  "UsePokeSnipers": true,
  "UsePokezz": true,
  "UsePokewatchers": true,
  "PokeSnipers2Exe": "",
  "RemoveAfter": 5,
  "ShowLimit": 1000,
  "VerifyOnSkiplagged": true,
  "UsePokemonGoIVClub": true,
  "UsePokeSnipe": true,
  "UseFilter": true,
  "UseGeoLocationBoundsFilter": false,
  "GeoLocationBounds": {
    "SouthWest": {
      "Latitude": 40.60414973724572,
      "Longitude": -74.2335024050293
    },
    "NorthEast": {
      "Latitude": 40.848719837349826,
      "Longitude": -73.660840051513674
    }
  },
  "AppTheme": "Dark",
  "IsServer": false,
  "IsManaged": true,
  "ServerHost": "pokefeeder.live",
  "ServerPort": 49000,
  "OutgoingServerPort": 49000,
  "ShareBotCaptures": true,
  "BotWebSocketPorts": [
    14251
  ],
  "VerifiedOnly": true,
  "MinimumIV": 0.0,
  "UnverifedOnly": false,
  "UseUploadedPokemon": true,
  "PokemonNotInFilterMinimumIV": 100.0
}`

**Comment by luigidanield2** [comment]
yes ok but  one configuration for dont riceve a permaban im a livel 29.....if i use pogo locations server i get a ban?????or no?????and if i use snipe at necrobot i riceve a ban????

**Comment by darkfriend77** [comment]
Hmmm ... not sure but it looks like it is only sniping pokemons from the list of "HumanWalkSnipeFilters"

and not the one from PokemonToSnipe ... ???

**Comment by iwean** [comment]
@darkfriend77 @luigidanield2 
Sorry for make confused, if you want pogolocfeeder function, first you must disable HumanWalk.
"Enable" = false

And if you not want to get ban, please do not use bot..... play legit.

**Comment by blueoceanvn** [comment]
It seems to work as @iwean suggested but I always get
[22:01:19] (SNIPER) Did not find a Pokemon within the Location, pokemon despawned?
Manually sniping it with pokersniper2 at the same location still find that same pokemon
